modules/imgproc/include/opencv2/imgproc.hpp

```diff
@@ -3902,6 +3902,51 @@ a filled ellipse sector is to be drawn.
 CV_EXPORTS_W void ellipse(InputOutputArray img, const RotatedRect& box, const Scalar& color,
                         int thickness = 1, int lineType = LINE_8);
 
+/* ----------------------------------------------------------------------------------------- */
+/* ADDING A SET OF PREDEFINED MARKERS WHICH COULD BE USED TO HIGHLIGHT POSITIONS IN AN IMAGE */
+/* ----------------------------------------------------------------------------------------- */
+
+//! Possible set of marker types used for the drawMarker function
```

Comment:
hi @StevenPuttemans , it will be nicer if you use cv::drawMarker like
//! Possible set of marker types used for the cv::drawMarker function